alma/alma-installments-prestashop
# stock subscription id

### Reason for change



[Linear task](https://linear.app/almapay/issue/ECOM-1215/store-the-subscription-in-bdd)

### Code changes



### How to test

- install a new clean infra
- install the insurance
- buy products with insurance
- in the bo list you will see now the subscription id, the status and the mode test/live
![image](https://github.com/alma/alma-installments-prestashop/assets/110386752/accb5906-d06c-485c-bda1-ea748a2f5bfc)
